1_turb_abl_fsi/template_files/nrel5mw_nalu.yaml

```diff
@@ -212,7 +212,7 @@ realms:
       - type: rotation
         angle: 5.0
         axis: [0.0, 1.0, 0.0]
-        centroid: [0.0, 0.0, 0.0]
+        centroid: [5.0, 0.0, 0.0]
```

Comment:
@psakievich Can we please switch transformations to - A value of `5` might be misleading to read, but will also change from one turbine to another. I imagine referencing these files for a long time to come.
      - type: translation
        displacement: [1795.0, 1800.0, 90.0]
      - type: rotation
        angle: 5.0
        axis: [0.0, 1.0, 0.0]
        centroid: [1800.0, 1800.0, 90.0]

Reply:
I was actually thinking the opposite for the same benefit. If we are going to position them with the driver files then we only have to change the position and won't have to change the centroid for each turbine we add.  I'm happy to oblige if that is still your preference though.

Reply:
Yes, it is more straightforward this way if we want to replicate. Although we should probably script this and other mesh transformations, strictly based on rotor and tower position. For now, can we add a comment next to this stating it is the horizontal distance between rotor center and tower center?

Reply:
Definitely.  I see what you are saying.